Core/Object Arts/Dolphin/Base/KernelLibrary.cls

```diff
@@ -1686,6 +1710,7 @@ zeroMemory: pvDest length: cBytes
 !KernelLibrary categoriesFor: #getTempFileName:lpPrefixString:uUnique:lpTempFileName:!public!win32 functions-file! !
 !KernelLibrary categoriesFor: #getTempPath:lpBuffer:!public!win32 functions-file! !
 !KernelLibrary categoriesFor: #getThreadTimes:lpCreationTime:lpExitTime:lpKernelTime:lpUserTime:!public!win32 functions-process and thread! !
+!KernelLibrary categoriesFor: #getTickCount!public!win32 functions-natural language support! !
```

Comment:
Indeed! I've selected more reasonable categories and committed a change. Now to see if it shows up as a new pull request...